### DISCUSSION TITLE
Can't get github education pack.. Is it a technical error?

### DISCUSSION THREAD

**Discussion by Aerish369** [community/community#117635] [GitHub Education] [answered]
### Hi everyone,

Hello everyone, 

I wanted to try github copilot. Since I am a student, I applied for the github student pack but in the upload proof section whenever I upload the proof of my academics in the college, it's refreshes and the page doesn't redirects anywhere. What should I do? Is my request recorded for further review??

![Screenshot from 2024-04-07 17-13-00](https://github.com/community/community/assets/107682299/f9c65ad7-569f-4549-ade4-a2b80e7c88ca)

**Comment by deXcripter** [comment, marked as answer]
Consider using a different device or switch to a different network. I had the same issue while opting for the student's package.
